src/core/os/amdgpu/amdgpuDevice.cpp

```diff
@@ -2119,6 +2123,23 @@ bool Device::HasFp16DisplaySupport()
     return supported;
 }
 
+// Returns true if gpu + amdgpu kms driver do support 16 bit unorm fixed point display.
+bool Device::HasRgba16DisplaySupport()
+{
+    bool supported = false;
+
+    // On Linux 5.14 (DRM 3.42) and later we also have the 64 bpp rgba16 unorm fixed point format
+    // on display engines of generation DCE 8.0 - DCE 12, and on all DCN engines, iow. Sea Islands
+    // and later.
+    if ((IsDrmVersionOrGreater(3, 42) || IsKernelVersionEqualOrGreater(5, 14)) &&
+        (IsGfx10Plus(m_chipProperties.gfxLevel) || IsGfx9(*this) || IsGfx8(*this) || IsGfx7(*this)))
```

Comment:
PAL no longer supports gfx7, we really should reference it anymore.

But, why not just do: (m_chipProperties.gfxLevel >= GfxIpLevel::GfxIp8)

Reply:
Regrettably, your driver doesn't support gfx7 anymore. Many of my users are still using pre-Polaris in production. This patch was written before AMD dropped pre-Polaris. I also left it in to allow easy back-porting to a final build of amdvlk by myself to provide my users on < Polaris, of which there are many, with this functionality.

I will switch to your proposed check. Thanks.

Reply:
great.